src/modules/removeItemFromList.js

```diff
@@ -0,0 +1,15 @@
+import { Store } from './index.js';
+
+class RemoveItemFromList {
+  static removeItemFromList = (itemList) => {
+    itemList.parentElement.remove();
+    const tasks = Store.getItems();
+    const newTasks = tasks.filter(
```

Comment:
- [ ] Good job up to this point. However, there is a project requirement you haven't met yet. Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..

![image](https://user-images.githubusercontent.com/98453979/199794195-9c30f32f-9248-4b6f-8ce8-18bf8bb5087b.png)